src/Enforcers/ModelValidationEnforcerTest.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+use PHPUnit\Framework\TestCase;
+
+use PDGA\DataObjects\Attributes\Column;
+use PDGA\DataObjects\Enforcers\ModelValidationEnforcer;
+use PDGA\DataObjects\Validators\EmailValidator;
+use PDGA\DataObjects\Validators\MaxLengthValidator;
+
+class NamedPerson
+{
+    #[Column(

```

Comment:
Can we get some whitespace separation between properties and attributes in this class?

Reply:
How would you like the spacing to be set?  A space between a property and the beginning of the next propertie's attributes? ie

`
    public int $id;

    #[Column(
        name: 'name',
        maxLength: 20
    )]
    #[MaxLengthValidator(15)]
    public string $name;
`